README.md

```diff
@@ -1,42 +1,160 @@
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-👤 **Author**
+# 📖 [Creative Commons Global Summit] <a name="about-project"></a>
+
+> 
+
+**[Creative Commons Global Summit]** Is a first capstone project in microverse, It includes all the lessons we learned in 4 weeks including HTML, javaScript, CSS, bootstrap 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">HTML</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Completed Mobile version]**
+- **[Completed Desktop version]**
+- **[Added functionalities on menu buttons for mobile version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://eddyshimwa.github.io/CapStone-Project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+### Usage
+
+
+
+### Run tests
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors 
+
+👤 **Author1**
 
 - GitHub: [@EddyShimwa](https://github.com/EddyShimwa)
 - Twitter: [@eddy56388406](https://twitter.com/eddy56388406)
-- LinkedIn: [Pray Eddy SHIMWA]([https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/]
+- LinkedIn: [Pray Eddy SHIMWA]([https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/])
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
 
+> Describe 1 - 3 features you will add to the project.
 
-## 🤝 Contributing
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
 
-Give a ⭐️ if you like this project!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-## 📝 License
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
```

Comment:
- According to the project requirement, let's gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). to make our readme more professional.